packages/gatsby-theme-styleguide/src/gatsby-components/Preview/PreviewFrame.js

```diff
@@ -111,48 +119,15 @@ export class PreviewFrame extends PureComponent {
     }
   `
 
-  componentDidMount() {
-    this.updateIframe()
-  }
-
-  updateIframe = () => {
-    const preview = this.props.data.node.codes
-    const iframe = this.refs.iframe
-    const document = iframe.contentDocument
-    const head = document.getElementsByTagName('head')[0]
-    const style = document.createElement('style')
-
-    if (preview.js) {
-      const js = document.createElement('script')
-      js.innerHTML = preview.js
-      head.appendChild(js)
-    }
-
-    if (preview.css) {
-      style.innerHTML = `
-        ${this.iframeCSS()}
-        ${preview.css}
-      `
-    } else {
-      style.innerHTML = this.iframeCSS()
-    }
-
-    head.appendChild(style)
-
-    document.body.innerHTML =
-      preview.html !== undefined ? preview.html : 'No html'
-    document.body.style.margin = 0
-
-    this.setState({
-      iframeHeight: this.refs.iframe.contentDocument.body.offsetHeight,
```

Comment:
keep this for iframes heights

Reply:
when switching to full screen and playing with viewport and restitching to normal => bug of iframe heights
=> recalculate height each time we change viewport as well as full screen